src/main/java/uk/gov/hmcts/reform/civil/handler/callback/camunda/notification/ClaimantResponseNotAgreedRepaymentDefendantLipNotificationHandler.java

```diff
@@ -61,32 +63,28 @@ public List<CaseEvent> handledEvents() {
 
     @Override
     public Map<String, String> addProperties(CaseData caseData) {
-
-        if (isRespondentNotRepresented(caseData)) {
+        if (caseData.isRespondent1NotRepresented()) {
             return Map.of(
                 CLAIM_REFERENCE_NUMBER, caseData.getLegacyCaseReference(),
                 DEFENDANT_NAME, getPartyNameBasedOnType(caseData.getRespondent1())
             );
+        } else {
+            return Map.of(
+                CLAIM_REFERENCE_NUMBER, caseData.getLegacyCaseReference(),
+                CLAIM_LEGAL_ORG_NAME_SPEC, organisationDetailsService.getRespondentLegalOrganizationName(caseData)
+            );
         }
-        return null;
-    }
-
-    private String addEmail(CaseData caseData) {
-        if (isRespondentNotRepresented(caseData)) {
-            return caseData.getRespondent1().getPartyEmail();
-        }
-        return null;
     }
 
     private String addTemplate(CaseData caseData) {
-        if (caseData.isRespondentResponseBilingual()) {
-            return notificationsProperties.getNotifyDefendantLipWelshTemplate();
+        if (caseData.isRespondent1NotRepresented()) {
+            if (caseData.isRespondentResponseBilingual()) {
+                return notificationsProperties.getNotifyDefendantLipWelshTemplate();
+            } else {
```

Comment:
again no need of nested if else if you are returning everywhere . also would be good to split in two methods : getTemplateForLipRespondent (and put the logic with biligual flag in there) so would be ```
if(caseData.isRespondent1NotRepresented()){
    return getTemplateForLipRespondent();
}
return notificationsProperties.getNotifyDefendantLrTemplate();
```

Reply:
not quite